### ISSUE TITLE
Individual Message Deletion - Not Working

### ISSUE THREAD

**Issue by HarbingerNight** [XtremeOwnage/WarBot#80]
**Describe the bug**
Individual Message Deletion not working. Have set to 180 min via website. In neither war or expedition are messages being deleted. Setting appears to be set still on website.

**To Reproduce**
1. Set war and expedition to method individual messages and time to 180 on website. 
2. Save.
3. Wait the appropriate amount of time after message is produced.

**Expected behavior**
The bot to delete its messages individually a set amount of time after they occure. 

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Ubuntu 22.04 WSL 2 on Windows 10
 - Browser: NA
 - Version:

**Smartphone (please complete the following information):**
 - Device: Huawei M5
 - OS: Android 8.0
 - Browser: 
   - Chrome
   - Termius SSL
   - Microsoft RD Client

**Additional context**
Hustle Prime ( ManDrako ADMIN )
![Screenshot_20220511-221441](https://user-images.githubusercontent.com/46084773/167991093-157f07cf-c68b-43b9-b6a8-75a6b1450b70.jpg)
![IMG_20220511_221347](https://user-images.githubusercontent.com/46084773/167991135-fd3f7bdf-f3f7-4e93-9713-e8be1a34031d.jpg)
![IMG_20220511_221324](https://user-images.githubusercontent.com/46084773/167991155-6446e6bb-d904-44aa-aa17-60d0de8c7531.jpg)

**Comment by HarbingerNight** [comment]
Just noticed this doesn't carry over to other wars/expedition and has to be set for each. Still doesn't seem to work for the one that was set. Will watch and update.

**Comment by XtremeOwnageDotCom** [comment]
I don't see any errors in the logs as I would expect-

I will note, I did not add any checks to validate the bot has the ability to both read messages for the channel, and delete messages for the channel for the clearing functionality. I opened issue #83 to add this in the future. 

But, until then, make sure WarBOT has both the ability to READ MESSAGES, as well as MANAGE MESSAGES for either guild-level, or individual channel level.
